### PR TITLE
[Zaraz] Supported tools page

### PR DESCRIPTION
**Pull request by marciocloudflare**
- Adds a page to Zaraz with all the supported tools.

### CODE REVIEW

File: content/zaraz/reference/supported-tools.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+--------------------------------- | -----------
+Bing                              | Advertising 
+Branch                            | Marketing automation
+Facebook Pixel                    | Advertising
+Floodlight                        | Advertising
+Google Ads                        | Advertising
+Google Analytics                  | Analytics
+Google Analytics 4                | Analytics
+Google Conversion Linker          | Miscellaneous
+Google Maps - Reserve with Google | Advertising  / Miscellaneous
```
Comment (pedrosousa) [inline content/zaraz/reference/supported-tools.md]: Nit: Some of the categories have two spaces before the slash.

File: content/zaraz/reference/supported-tools.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+pcx-content-type: reference
+title: Third-party tools
+meta:
+    title: Third-party tools supported by Cloudflare Zaraz
+---
+
+# Third-party tools supported
```
Comment (pedrosousa) [inline content/zaraz/reference/supported-tools.md]: `Supported third-party tools` sounds slightly better. Wdyt?

File: content/zaraz/reference/supported-tools.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Bing                              | Advertising 
+Branch                            | Marketing automation
+Facebook Pixel                    | Advertising
+Floodlight                        | Advertising
+Google Ads                        | Advertising
+Google Analytics                  | Analytics
+Google Analytics 4                | Analytics
+Google Conversion Linker          | Miscellaneous
+Google Maps - Reserve with Google | Advertising  / Miscellaneous
+HubSpot                           | Marketing Automation
```
Comment (pedrosousa) [inline content/zaraz/reference/supported-tools.md]:
```suggestion
HubSpot                           | Marketing automation
```

File: content/zaraz/reference/supported-tools.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+Snapchat                          | Advertising
+Taboola                           | Advertising
+Tatari                            | Advertising
+Twitter Pixel                     | Advertising
+Upward                            | Recruiting
+ZipRecruiter                      | Recruiting
+
+{{</table-wrap>}}
+
+For all tools not in the above list, use the custom integrations below:
```
Comment (pedrosousa) [inline content/zaraz/reference/supported-tools.md]:
```suggestion
For any other tool, use the custom integrations below:
```
I believe this version would be easier to read.

**Review by pedrosousa — APPROVED** [review]

File: content/zaraz/reference/supported-tools.md (hunk 2 above)
Comment (marciocloudflare) [reply content/zaraz/reference/supported-tools.md]: Agree!

File: content/zaraz/reference/supported-tools.md (hunk 1 above)
Comment (marciocloudflare) [reply content/zaraz/reference/supported-tools.md]: On it

**Pull request merged**